tests/NHSD.BuyingCatalogue.Ordering.Domain.UnitTests/OrderTests.cs

```diff
@@ -375,5 +375,71 @@ public void CalculateCostPerYear_OneOff_OrderItemCostTypeRecurring_ReturnsZero()
 
             order.CalculateCostPerYear(CostType.OneOff).Should().Be(0);
         }
+
+        [TestCase(nameof(CatalogueItemType.AssociatedService), nameof(ProvisioningType.OnDemand), 10, 2, 720)]
+        [TestCase(nameof(CatalogueItemType.AssociatedService), nameof(ProvisioningType.Declarative), 20, 4, 960)]
+        public void CalculateTotalOwnershipCost_SingleOneOffOrRecurringOrderItem_ReturnsTotalOwnershipCost(string catalogueItemTypeName, string provisioningTypeName, decimal price, int quantity, decimal totalOwnershipCost)
```

Comment:
Nice 👍

Reply:
Aw man I thought you were doing another PR comment then 😆